src/Templates/TemplateSolutions/CS.CustomTypeEditor/CustomControlLibrary.Client/CustomControlLibrary.Client.csproj

```diff
@@ -5,12 +5,9 @@
     <UseWindowsForms>true</UseWindowsForms>
     <LangVersion>9.0</LangVersion>
     <Nullable>enable</Nullable>
+    <UseDesignerSdk>true</UseDesignerSdk>
```

Comment:
```suggestion
    <UseDesignerSDK>true</UseDesignerSDK>
```

Reply:
Only 2 letter acronyms are all caps, 3 letter ones follow capitalization of the word